test/config.test.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Test config.js parameters for all providers
+ *
+ * @author: Drazen Urch
+ */
+
+import each from "jest-each";
+
+const workflow = workflowSet();
+
+const PPConfigs = workflow.__get__("PPConfigs");
+const getConfigId = workflow.__get__("getConfigId");
+
+let activeConfig = () => PPConfigs()[getConfigId()];
+
+each(PPConfigs().filter((config) => config.id > 0).map((config) => [config.id]))
+    .describe("CONFIG_ID = %i", (configId) => {
+        beforeEach(() => {
+            workflow.__set__("CONFIG_ID", configId);
+        });
+
+        test("ensure `get-more-passes-url` is a valid URL", () => {
+            new URL(activeConfig()['get-more-passes-url']);
+        })
```

Comment:
Maybe we could add a test to ensure that the URL is fixed to the correct value for both configs? I doubt the CF value may change, but not sure if this is also the case for HC.